src/routes/(admin)/tests/test-[type]/[id]/+page.svelte

```
@@ -45,12 +46,29 @@
+	});
+
 	$effect(() => {
 		if (data.convertTemplate && data.testData) {
 			currentScreen = typeOfScreen.questions;
 		}
 	});
 
+	$effect(() => {
+		if (convertTemplate && currentScreen === typeOfScreen.primary) {
+			if (page.url.searchParams.has('template_id')) {
+				goto('?', { invalidateAll: true, noScroll: true, keepFocus: true });
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)